src/main/java/entities/GameAccess.java

```diff
@@ -117,11 +117,6 @@ public interface GameAccess {
      */
     boolean isValidCard(Card card);
 
-    /**
-     * Notify all GameObservers observing this Game that there has been a change.
-     */
-    void notifyGameObservers();
-
```

Comment:
Does this method need to be replaced anywhere, or was notifying game observers already updated in an earlier PR?

Reply:
I believe there is a notifier class whose purpose is to notify observers.